### PR TITLE
updated user messages & UI with correct spelling

### PR DESCRIPTION
**Pull request by amitamir**

### CODE REVIEW

**Comment by amitamir** [comment]
not sure if branch is updated with latest commits from Releases/Official-Release

File: Ginger/GingerCoreTest/UITests/PBDriverWidgetTest.cs [hunk 1]
```diff
@@ -137,7 +137,7 @@ public static void ClassInit(TestContext context)
 
             c = new ActSwitchWindow();
             c.LocateBy = eLocateBy.ByTitle;
-            c.LocateValueCalculated = "CSM Widgets Test Applicaiton";            
+            c.LocateValueCalculated = "CSM Widgets Test Application";
```
Comment (Maheshkale447) [inline Ginger/GingerCoreTest/UITests/PBDriverWidgetTest.cs:140]: Please check if modified unit test cases are working fine

File: Ginger/Ginger/ApplicationModelsLib/ModelOptionalValue/AddModelOptionalValuesWizard.cs [hunk 2]
```diff
@@ -34,7 +34,7 @@ public enum eSourceType
             Excel,
             [EnumValueDescription("XML")]
             XML,
-            [EnumValueDescription("Json")]
+            [EnumValueDescription("JSON")]
```
Comment (Maheshkale447) [inline Ginger/Ginger/ApplicationModelsLib/ModelOptionalValue/AddModelOptionalValuesWizard.cs:37]: Please check if solution from previous version is loading with old enum value

File: Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/AddAPIModelWizard.cs [hunk 3]
```diff
@@ -40,7 +40,7 @@ public enum eAPIType
             WSDL,
             [EnumValueDescription("XML Templates")]
             XMLTemplates,
-            [EnumValueDescription("Json Templates")]
+            [EnumValueDescription("JSON Templates")]
```
Comment (Maheshkale447) [inline Ginger/Ginger/ApplicationModelsLib/APIModels/APIModelWizard/AddAPIModelWizard.cs:43]: Please check if solution from previous version is loading with old enum value

**Review by Maheshkale447 — CHANGES_REQUESTED** [review]

**Review by Maheshkale447 — APPROVED** [review]

**Pull request merged**